demisto_sdk/commands/init/contribution_converter.py

```diff
@@ -660,13 +664,13 @@ def create_pack_base_files(self):
         to be in the base directory of a pack
         """
         logger.info("Creating pack base files")
-        fp = open(os.path.join(self.pack_dir_path, "README.md"), "a")
+        fp = open(os.path.join(self.working_dir_path, "README.md"), "a")
         fp.close()
```

Comment:
```suggestion
        Path(self.working_dir_path, "README.md").touch()
```
Modify the other two the same way as well, please